collections/Bodl/MS_Bodl_851.xml

```diff
@@ -37,124 +37,780 @@
                      <idno type="SCN">3041</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Walter Map, <title>De nugis curialium</title>; Latin poetry; <title>Piers
+                     Plowman</title> (Z- and C-texts); England, 14th century, late, or 15th century,
+                  early, with additions</head>
                <physDesc>
-                  <p n="composite">Composite: two parts</p>
+                  <p>A manuscript of several sections: three (fols. 7-77, 78-123, 124-139) from the
+                     late fourteenth or early fifteenth century, essentially uniform in decoration,
+                     and with common scribes; supplemented in the fifteenth century by fols.
+                     140-208, and with additions (fols. 1-6 and elsewhere) by 'Dodsthorp' and another contemporary scribe made
+                     at Ramsey Abbey at some point after 1405.
+                     <!--See Rigg (1994), pp. 25-7 for a resconstruction of the sequence of production.--></p>
+                  <objectDesc>
+                     <supportDesc>
+                        <extent> 207 (6 + 71 + 46 + 16 + 68) + 2 leaves <dimensions type="leaf"
+                              unit="mm">
+                              <height max="250" min="245">245-250</height>
+                              <width min="175" max="180">175-180</width>
+                           </dimensions> Quire 14 slightly smaller than the rest of the volume. </extent>
+                        <foliation>1-208 in early-modern ink, skipping 166; 210 and 209 (in that
+                           order) in later pencil</foliation>
+                        <collation>For the collation of the individual sections see below. The
+                           quires were numbered i-[xxiii] on the final verso, bottom centre, when
+                           the volume was put together in the fifteenth century.</collation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding notAfter="1600" notBefore="1500">
+                        <p>16th(?)-century binding of plain brown leather over pasteboards;
+                           rebacked.</p>
+                     </binding>
+                  </bindingDesc>
+                  <accMat>The front pastedown is a fragment of a 15th-century (?) Ramsey abbey
+                     account: John Alb... reeve, William atte Cros late reeve; tenants include
+                     William atte Castel, Ralph Freman.</accMat>
                </physDesc>
+               <history>
+                  <provenance><persName role="fmo" key="person_3464">John Wells, monk of
```

Comment:
There is some inconsistency in what we should include in `<persName>` – dates? title? name only?